department-of-veterans-affairs/caseflow
# Add a review and confirmation page for substitution

Resolves [Jira issue 1082](https://vajira.max.gov/browse/CASEFLOW-1082)

### Description

This PR adds a review and confirm page where the user can confirm that they have selected a substitute appellant.  The user will be able to see the appellant’s information and information the tasks identified for creation.   This PR is dependent on PR #16192 

Testing Plan
- Navigate to a dispatched appeal where the appeal’s veteran has a date of death
- Create a substitute appeal and ensure that the new appellant’s information (name, relationship, and substitution date) display correctly
- Also, ensure that all automated tests pass


### Acceptance Criteria
- [ ] the user is able to see the following on the confirmation page: 
- Appellant Name 
- Relationship to Veteran
- Granted Substitution Date 
- Tasks identified for creation (if applicable)
- [ ] if an evidence submission task was selected, the window end date should be calculated with the following formula
- Original Evidence Submission Window End Date (actual) - Appellant Date of Death = number of days remaining + Granted Substitution Date = substitution evidence submission end date

### Testing Plan
- Navigate to a dispatched appeal where the appeal’s veteran has a date of death
- Create a substitute appeal and ensure that the new appellant’s information (name, relationship, and substitution date) display correctly
- Also, ensure that all automated tests pass
